.github/pull_request_template.md

```diff
@@ -14,4 +14,5 @@ Closes
 
 - [ ] Remove debugging artifacts (if needed)
 - [ ] Rebuild the Setup_Hive.exe
-- [ ] Update Hive Wiki (if needed)
\ No newline at end of file
+- [ ] Update Hive Wiki (if needed)
+- [ ] Update most important Grasshopper template(s); as of now C:\Users\Christoph\Documents\GitHub\hive\GrasshopperExamples\LectureExercises\EaCS3_E04_Hive_Template.gh
```

Comment:
ups I mean `..\hive\GrasshopperExamples\LectureExercises\EaCS3_E04_Hive_Template.gh`

Reply:
changed it to the github and relativepath